backend/src/apiserver/model/resource_reference.go

```diff
@@ -27,6 +27,9 @@ type ResourceReference struct {
 	// The ID of the resource that been referenced to.
 	ReferenceUUID string `gorm:"column:ReferenceUUID; not null; "`
 
+	// The name of the resource that been referenced to.
+	ReferenceName string `gorm:"column:ReferenceName; not null; "`
```

Comment:
Adding a new required field usually breaks backwards compatibility with all existing clients.
What do you think about keeping it optional?

Reply:
This is database schema requirement which won't be surfaced through UI. Please see integration tests for the sample request.